### PR TITLE
Update SDK docs for SDK v5.2.2 - Closes #1460

### LINKED ISSUE #1460: Update SDK docs to 5.2.2
### Which content is outdated?
- Version 5.2.1 
- Node.JS version 12
- Lisk Commander 5.1.7
- lisk-client 5.2.1

### What should be the new content?
- Version 5.2.2
- Node.JS version 16
- Lisk Commander 5.1.10
- lisk-client 5.2.2

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1460

### CODE REVIEW

File: modules/ROOT/pages/quickstart.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you are using NVM, (Node.js Version Manager), ensure you install the correct
 
 [source,bash]
 ----
-nvm install v12.22.9
+nvm install v16.15.0
 ----
 
 NOTE: The correct version for NPM(6) is installed automatically with version 12 of Node.js.
```
Comment (chris529) [inline modules/ROOT/pages/quickstart.adoc]:
```suggestion
NOTE: The latest version for NPM(6) is installed automatically with version 12 of Node.js.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just 1 small point 👍

File: modules/ROOT/pages/quickstart.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/quickstart.adoc]:
```suggestion
```

File: antora.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: lisk-sdk
 title: Lisk SDK
 version: 'master'
```
Comment (ishantiw) [inline antora.yml:3]:
```suggestion
version: 'main'
```

**Review by ishantiw — APPROVED** [review]
LGTM, just a minor comment

**Pull request merged**